cheatsheets/Password_Storage_Cheat_Sheet.md

```diff
@@ -138,11 +138,11 @@ PBKDF2 requires that you select an internal hashing algorithm such as an HMAC or
 
 The work factor for PBKDF2 is implemented through an iteration count, which should set differently based on the internal hashing algorithm used.
 
-- PBKDF2-HMAC-SHA1: 720,000 iterations
-- PBKDF2-HMAC-SHA256: 310,000 iterations
-- PBKDF2-HMAC-SHA512: 120,000 iterations
+- PBKDF2-HMAC-SHA1: 1,300,000 iterations
```

Comment:
Are these for privileged or non-privileged accounts? I vaguely recall NIST having 2 different workfactors for these two categories of users.

Reply:
Can you please give link of any resource that explains about privileged or non-privileged accounts, under NIST work factors.